builtin/logical/pki/backend.go

```diff
@@ -460,6 +468,25 @@ func (b *backend) invalidate(ctx context.Context, key string) {
 		b.crlBuilder.markConfigDirty()
 	case key == storageIssuerConfig:
 		b.crlBuilder.invalidateCRLBuildTime()
+	case strings.HasPrefix(key, crossRevocationPrefix):
+		split := strings.Split(key, "/")
+
+		if !strings.HasSuffix(key, "/confirmed") {
+			cluster := split[len(split)-2]
+			serial := split[len(split)-1]
+			// Only process confirmations on the perf primary.
+			b.crlBuilder.addCertForRevocationCheck(cluster, serial)
+		} else {
+			if len(split) >= 3 {
+				cluster := split[len(split)-3]
+				serial := split[len(split)-2]
+				if !isNotPerfPrimary {
+					b.crlBuilder.addCertForRevocationRemoval(cluster, serial)
+				}
+			}
+		}
+
+		b.Logger().Debug("got replicated cross-cluster revocation: " + key)
```

Comment:
@stevendpclark Had we reached a consensus around adding or not adding debug logs about X-cluster replication related events?